invenio_administration/assets/semantic-ui/js/invenio_administration/src/search/SearchResults.js

```diff
@@ -34,7 +34,11 @@ export const SearchResults = ({ paginationOptions, currentResultsState }) => {
           <Grid.Column width={4}>
             <Count
               label={() => (
-                <>{i18next.t("{{count}} results found", { count: total })}</>
+                <>
+                  {i18next.t("{{count}} results found", {
+                    count: total.toLocaleString("en-US"),
```

Comment:
it hardcodes the en-US notation, unfotunately it won't work if someone has different locale

Reply:
Currently this is standard practise :( https://github.com/search?q=%28org%3Ainveniosoftware+OR+org%3Azenodo%29+.toLocaleString(%22en-US%22)&type=code